docs/photoniq/vwrs/waitingroom.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Waiting Room HTML page
+---
```

Comment:
This page needs an intro paragraph or two explaining what the HTML for and what it is about.